.eslintrc.json

```diff
@@ -0,0 +1,35 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "jest": true
+    },
+    "parser": "babel-eslint",
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "extends": [
+        "airbnb-base"
+    ],
+    "rules": {
+        "no-shadow": "off",
+        "no-param-reassign": "off",
+        "eol-last": "off",
+        "arrow-parens": "off",
+        "no-tabs": 0,
+        "no-unused-vars": "off",
+        "import/prefer-default-export": "off",
+        "no-case-declarations": "off",
+        "no-use-before-define": "off",
+        "default-case": "off",
+        "no-undef": "off",
+        "import/no-cycle": "off",
+        "consistent-return": "off",
+        "no-const-assign": "off"
+    },
+    "ignorePatterns": [
```

Comment:
- [ ] Please update these eslint rules to reflect only the rules listed [here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json). It's important you use only the rules shared by other students. It improves the professionalism of the project.